module/apmecho/v4/example_test.go

```diff
@@ -0,0 +1,39 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package apmecho_test
+
+import (
+	"github.com/labstack/echo/v4"
```

Comment:
Because of how echo adds the version in the module definition, rather than as a directory, this package won't build with Go 1.8 (see https://travis-ci.org/elastic/apm-agent-go/jobs/507736180).

Can you please add a build constraint for 1.9+ to each of the Go files in this new package? Look for "// +build go1.9" in the files in apmgocql for an example, if you're unfamiliar with doing that.

Reply:
That is an awesome idea 👍 I added the build tags in `v4` files.